pkg/parser/parser.y

```diff
@@ -1093,6 +1093,8 @@ import (
 	AdminStmtLimitOpt                      "Admin show ddl jobs limit option"
 	AllOrPartitionNameList                 "All or partition name list"
 	AlgorithmClause                        "Alter table algorithm"
+	AlterJobOptionList 				       "Alter job option list"
+	AlterJobOption 				           "Alter job option"
```

Comment:
Could you run `make fmt` in the parser directory?

Reply:
updated in [c75fe5c](https://github.com/pingcap/tidb/pull/57268/commits/c75fe5c329c3f106c474a2febdb8d2afc2d76c61)